lms/djangoapps/verify_student/models.py

```diff
@@ -47,7 +47,7 @@
 
 def generateUUID():  # pylint: disable=invalid-name
     """ Utility function; generates UUIDs """
-    return str(uuid.uuid4())
+    return six.text_type(uuid.uuid4())
```

Comment:
It may not cause any problems, but this changes the return type from bytes to text.

Reply:
I think this is probably okay.